tools/ResourceMetaSchema.json

```diff
@@ -204,6 +204,32 @@
         {
           "type": "object",
           "properties": {
+            "anyOf": {
```

Comment:
That's right. We encourage people to use "oneOf" when possible. This time we have a special case for Microsoft.Compute. They need "anyOf" to describe their extensions.